build_tools/web/flutter.js.patch

```diff
@@ -0,0 +1,19 @@
+--- flutter.js	2023-03-24 18:27:43
```

Comment:
// This script is needed to patch the flutter.js and flutter_service_worker.js so they load faster 
// see: https://github.com/dwyl/app/issues/326#issuecomment-1478314967